.github/workflows/build.yml

```diff
@@ -8,6 +8,10 @@ on:
     types: [apb]
 
 env:
+  AWS_DEFAULT_REGION: us-east-1
+  BUILD_BUCKET: ${{ secrets.THIRD_PARTY_BUCKET_STAGING }}
```

Comment:
I see that we added this `BUILD_BUCKET` env variable, but I don't see us using it anywhere.

Reply:
Isn't it used in `src/assessor_tools.yml` and `src/cobalt_strike.yml`?

I ran this command to determine where it was needed:
```
find src -name "*.yml" -exec grep -HniF build_bucket {} \;
```

Reply:
You are correct- I am mistaken here.  Never mind! 🤦